UM/Qt/qml/UM/Dialog.qml

```diff
@@ -97,6 +99,14 @@ Window
         base.visible = true;
     }
 
+    onVisibleChanged:
+    {
+        if(selfDestroy && !visible)
+        {
+            destroy()
```

Comment:
We're not too consistent with it, but since this is more than a one-liner and thus 'in' javascript, we should add the `;` after `destroy()` maybe.

Reply:
Right, I do prefer to add it but since our QML part comes with Python code, I often forget :smile: